40-PicoSearch.php

```diff
@@ -64,7 +64,7 @@ public function onRequestFile(&$file)
                 $folder = preg_replace('~\.+/~', '', $folder);
             }
 
-            $temp_file = $pico->getConfig('content_dir') . ($folder ?: '') . 'search' . $pico->getConfig('content_ext');
+            $temp_file = $pico->getConfig('content_dir').$folder.'search'.$pico->getConfig('content_ext');
```

Comment:
Could you add back the spaces around the `.` operators, for consistency with the rest of the code?